lib/linguist/languages.yml

```diff
@@ -2176,6 +2176,16 @@ Git Attributes:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 956324166
+Git Blame Ignore:
+  type: data
+  color: "#F44D27"
+  aliases:
+  - Git Blame Ignore Revs
+  filenames:
+  - ".git-blame-ignore-revs"
+  tm_scope: text.git-blame-ignore
```

Comment:
**Nit:** The `scopeName` should (ideally) be prefixed by `source` instead of `text`.

Rationale: Generally speaking, `prose` and `markup`-type languages use grammars with scopeNames beginning with `text`, whereas `programming` and `data` languages are left with `source` instead.